### PR TITLE
created global navbar

### LINKED ISSUE #8: FD-003.1 Create Global Nav Bar 🌎
**Description:**

- Create the Navbar with the appropriate tabs and YNA logo.

**Acceptance** Criteria:

-  Tabs:

   -   Home | About | Members | Contact | Join

   - All links should redirect to the appropriate pages. 

   - Add testing component to check for functionality. 

**Figma:**

- https://www.figma.com/file/nrf9QHTznYOpjBw8yGXD4t/YNA-Website?node-id=156%3A12

### PR DESCRIPTION
**Pull request by Chaitanya-087**
global navbar created and demo components are updated to closes #8

### CODE REVIEW

**Comment by Chaitanya-087** [comment]
image

**Comment by anujverma-eng** [comment]
@Chaitanya-087 Kindly check [this](https://github.com/Young-N-Ambitious/yna_frontend/pull/17),
we have already added the Routes here!!
Do make your change in code, accordingly.

**Comment by Chaitanya-087** [comment]
@anujverma-eng  updated the folder structure and routes
Thank's for notifying

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-import { Home } from './pages/homePage';
-import { About } from './pages/aboutUs';
-import { Members } from './pages/members';
-import { Contact } from './pages/contactUs';
-import { Join } from './pages/joinUs';
-import { Login } from './pages/login';
+import './App.css';
+import {BrowserRouter, Routes, Route} from 'react-router-dom'
+import {HomePage, AboutUs, ContactUs, JoinUs, Members, Login, NotFound} from './pages'
+import {NavBar} from './pages/globalComponents'
```
Comment (anujverma-eng) [inline src/App.js:4]: @Chaitanya-087 Kindly Import Navbar from './pages' only,
as you can export this NavBar from './pages/index.js'

File: src/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import HomePage from './homePage/HomePage'
+import AboutUs from './aboutUs/AboutUs'
+import ContactUs from './contactUs/ContactUs'
+import JoinUs from './joinUs/JoinUs'
+import Members from './members/Members'
+import Login from './login/login'
+import NotFound from './notFound/NotFound'
+
```
Comment (anujverma-eng) [inline src/pages/index.js:8]: use Navbar here as:
import {Navbar} from './globalComponents'

and then export it.

**Review by anujverma-eng — CHANGES_REQUESTED** [review]
@Chaitanya-087  I've added a few comments, do check them out.

**Review by SincerelyBrittany — CHANGES_REQUESTED** [review]
The routes were already added. The navbar should only need to import the routes. I would recommend waiting until https://github.com/Young-N-Ambitious/yna_frontend/pull/17 is merged

**Comment by Chaitanya-087** [comment]
Okk

**Comment by Sydnee510** [comment]
hello @Chaitanya-087 can you resolve merge conflicts so we can go ahead and get this approved

**Comment by lanicemp** [comment]
Im confused on the status of this PR.  I dont see any files changed.  What are the next steps @Sydnee510 @SincerelyBrittany

**Pull request closed without merging**